documentation/src/markdown/DebuggingSSL.md

```diff
@@ -0,0 +1,75 @@
+<!--- Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com> -->
+# Debugging SSL Connections
+
+In the event that an HTTPS connection does not go through, debugging JSSE can be a hassle.
+
+WS SSL provides configuration options that will turn on JSSE debug options defined in the [Debugging Utilities](https://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/JSSERefGuide.html#Debug) and  [Troubleshooting Security](https://docs.oracle.com/javase/8/docs/technotes/guides/security/troubleshooting-security.html) pages.
+
+To configure, set the `play.ws.ssl.debug` property in `application.conf`:
+
+```
+play.ws.ssl.debug = {
```

Comment:
We'll need to decide how to document and put those where.

For akka we do `akka.ssl-config` as overrides and it falls back to `ssl-config` keys if not overrides there, which are the defaults from this lib then.